src/textual/widgets/_tabbed_content.py

```diff
@@ -430,13 +432,19 @@ def add_pane(
         pane = self._set_id(pane, tabs.tab_count + 1)
         assert pane.id is not None
         pane.display = False
+
+        async def _add_part(awaitable: AwaitComplete | AwaitMount) -> None:
```

Comment:
This doesn't seem right. If you need to wrap those awaitables in a coroutine, it suggests that maybe `AwaitComplete` is type incorrectly.

Maybe `AwaitComplete` needs to accept an `Awaitable`?

Reply:
So the code was wrong anyway, you mean, and the hint tweak is moot? If so I'll close this. Was just a passing tidy-up anyway.

Reply:
It may be correct, but the typing error stems from `AwaitComplete`

Reply:
Okay. I'll close this and flag up that AwaitComplete might need a wee bit more tinkering.